Makefile

```diff
@@ -15,7 +15,7 @@ makel.mk:
 		curl \
 		--fail --silent --show-error --insecure --location \
 		--retry 9 --retry-delay 9 \
-		-O https://gitea.petton.fr/DamienCassou/makel/raw/tag/v0.8.0/makel.mk; \
+		-O https://raw.githubusercontent.com/DamienCassou/makel/master/makel.mk; \
```

Comment:
Not sure what happened here. I see the following in my local repo.

````
-O https://github.com/DamienCassou/makel/raw/tag/v0.8.0/makel.mk; \
````

Reply:
I was experimenting because the CI was failing. I know understand the problem:

> https://github.com/DamienCassou/makel/raw/tag/v0.8.0/makel.mk

is invalid but

> https://github.com/DamienCassou/makel/raw/v0.8.0/makel.mk;

is ok. I thus changed your commit to have the correct URL.